src/main/java/net/minecraftforge/fml/DistExecutor.java

```diff
@@ -46,7 +46,7 @@
     private DistExecutor() {}
 
     /**
-     * Run the callable in the supplier only on the specified {@link Side}.
+     * Run the callable in the supplier only on the specified {@link Dist}.
```

Comment:
Please leave this class alone, it is not in the scope of this PR

Reply:
Is it worth splitting it off into its own PR?